pom.xml

```diff
@@ -105,6 +105,40 @@
             <version>1.9.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
```

Comment:
In fact, workflow-step-api dependency is inherited from workflow-durable-task-step and and some test scopes were set up